src/Callbacks/RequestSession.php

```diff
@@ -35,7 +35,16 @@ public function __construct(ResolverInterface $resolver)
      */
     public function __invoke(Report $report)
     {
-        if ($data = $this->resolver->resolve()->getSession()) {
+        $request = $this->resolver->resolve();
+        $data = null;
+
+        try {
+            $data = $request->getSession();
```

Comment:
Instead, update the getSession method so that it does not throw an exception please.